### PR TITLE
NO-ISSUE: Disable SNO jobs for ocm-2.10 branch

### PR DESCRIPTION
**Pull request by danmanor**

### CODE REVIEW

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10-presubmits.yaml [hunk 1]
```diff
@@ -1015,7 +1015,6 @@ presubmits:
       job-release: "4.15"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-assisted-service-release-ocm-2.10-e2e-ai-operator-ztp-sno-day2-masters
-    optional: true
```
Comment (adriengentil) [inline ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10-presubmits.yaml:1018]: these jobs still need to be optional, no?

Comment (danmanor) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10-presubmits.yaml:1018]: What is the point of having an optional job that is not running unless you trigger it ?

Comment (adriengentil) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10-presubmits.yaml:1018]: without optional=true, you'll need to trigger the job manually to make the check pass

Comment (danmanor) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10-presubmits.yaml:1018]: You mean without `optional=true`, if you choose to trigger the job and it failed you will have to override it in order to merge ?

**Comment by danmanor** [comment]
@adriengentil I changed it

**Comment by adriengentil** [comment]
> @adriengentil I changed it
but is this what we want? To trigger it manually to make the PR check pass and merge?

**Comment by danmanor** [comment]
@adriengentil Are you sure this is the case here? I was under the impression that job not triggered automatically will never be required

**Pull request closed without merging**